src/main/java/com/puppycrawl/tools/checkstyle/JavadocDetailNodeParser.java

```diff
@@ -88,12 +94,42 @@
     private DescriptiveErrorListener errorListener;
 
     /**
+     * When this field is set true, if the javadoc has non-tight HTML then that shall be reflected
+     * by the {@link ParseStatus} object returned by the method
+     * {@link #parseJavadocAsDetailNode(DetailAST, boolean)}
+     * through {@link ParseStatus#parseErrorMessage}.
+     *
+     * @see AbstractJavadocCheck#violateExecutionOnNonTightHtml
+     * @see AbstractJavadocCheck#doesCheckAcceptJavadocWithNonTightHtml()
+     * @see <a href="http://checkstyle.sourceforge.net/writingjavadocchecks.html#Tight-HTML_rules">
+     *     Tight HTML rules</a>
+     */
+    private boolean logViolationForNonTightHtmlInJavadoc;
```

Comment:
We can do parameter passing from `parseJavadocAsDetailNode` to `parseJavadocAsParseTree` and avoid the global variable ? Would you like it to be done that way ?